UI/idian/obs-propertieslist.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <QFrame>
+#include <QWidget>
+#include <QLayout>
+
+class OBSActionBaseClass;
+
+class OBSPropertiesList : public QFrame {
+	Q_OBJECT
+
+public:
+	OBSPropertiesList(QWidget *parent = nullptr);
+
+	void addRow(OBSActionBaseClass *ar);
+	void clear();
+
+	QList<OBSActionBaseClass *> rows() const { return rowsList; }
+
+private:
+	QVBoxLayout *layout;
+	QList<OBSActionBaseClass *> rowsList;
+};
```

Comment:
Example:
```suggestion
class OBSPropertiesList : public QFrame {
	Q_OBJECT

	QVBoxLayout *layout;
	QList<OBSActionBaseClass *> rowsList;
public:
	OBSPropertiesList(QWidget *parent = nullptr);

	void addRow(OBSActionBaseClass *ar);
	void clear();

	QList<OBSActionBaseClass *> rows() const { return rowsList; }
};
```

Reply:
OBS is pretty inconsistent with it, but it's a stylistic choice that you'll also find in Qt to have an explicit `private:` section and order declarations in ascending order of privacy (i.e. public -> protected -> private).